src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +51,10 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> person.getSex().equals(Person.Sex.MAN)
+                        && person.getAge() >= fromAge && person.getAge() <= toAge)
```

Comment:
```suggestion
                .filter(person -> person.getSex().equals(Person.Sex.MAN)
                        && person.getAge() >= fromAge && person.getAge() <= toAge)
```
If you have a long boolean expression you may create your own implementation of Predicate and use it in your filter. - це з рекомендацій до завдання. Я б виносив цю умову в Predicate бо вважаю її довгою, а ти вирішуй сам

Reply:
виніс умову в окремий предикат